client/blocks/subscription-length-picker/option.jsx

```diff
@@ -140,13 +140,13 @@ export class SubscriptionLengthOption extends React.Component {
 		const { term, translate } = this.props;
 		switch ( term ) {
 			case TERM_BIENNIALLY:
-				return translate( '%s year', '%s years', { count: 2, args: '2' } );
+				return translate( '%s year', '%s years', { count: 2, args: '2', context: 'subscription length' } );
```

Comment:
Question... Why is this line 2/2 and the others are 1/1?

Out of curiosity, why is count a number, and args a string?

Reply:
`count` is used to select the right plural per language (like the third argument for `_n()` in PHP), `args` is to substitute the variable in. The reason it's a string is to allow for formatting using dots and commas. I agree that this doesn't make that much sense in this case, but this is what it is in Core, and we use this here too.

Reply:
So in PHP this'd be `sprintf( _n( '%s year', '%s years', 2 ), '2' );`

Reply:
Aaaaah, ok! :) Get it now, thank you! 🙇